Samples/Excel-Get-Started-With-Dev-Kit/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "office-addin-taskpane-js",
+  "version": "0.0.1",
```

Should we mark this as a release version?
```suggestion
  "version": "1.0.0",
```